includes/amp-helper-functions.php

```diff
@@ -1,52 +0,0 @@
-<?php
-
-function amp_get_permalink( $post_id ) {
```

Comment:
We need to keep these global functions around for back-compat. They can just be stubs that call the new methods.